api-reference/10 UI Components/dxChat/1 Configuration/reloadOnChange.md

```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to call [reload()](/Documentation/ApiReference/Data_Layer/DataSource/Methods/#reload) on data source when the data changes.
```

Comment:
I'm not sure I understand the phrase "call reload() on data source". Judging by the link, the DataSource object should have a reload() function. 

But then I'm not sure what you mean by "data changes" and what "reload()" does. Does data change in the data source? Or does data change in the component? 

The best I can say is this... but that may not be easily understood, because there's no hint on what caused the data to change. Does someone write something to the data source outside of the current app? Or does this data change happen through the Chat component? I think it makes sense to clarify this. 

```suggestion
Specifies whether to call the data source's [reload()](/Documentation/ApiReference/Data_Layer/DataSource/Methods/#reload) function when data changes.
```